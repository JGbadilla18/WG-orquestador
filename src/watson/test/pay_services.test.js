const testWatson = require('./testWatson');

watsonText = 'Pagos servicio'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});

watsonText2 = 'Pagos servicio app'

it(watsonText2, () => {
  expect.assertions(1);
  return testWatson(watsonText2).then(data => expect(data).toMatchSnapshot());
});

watsonText3 = 'Pagos servicio web'

it(watsonText3, () => {
  expect.assertions(1);
  return testWatson(watsonText3).then(data => expect(data).toMatchSnapshot());
});