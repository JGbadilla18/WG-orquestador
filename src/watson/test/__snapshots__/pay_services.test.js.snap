// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagos servicio 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Conoce cómo pagar tus servicios de forma inmediata como agua, luz , teléfono y a más de 2, 300 proveedores como universidades, colegios o mantenimiento del hogar.",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": " https://www.corporacionbi.com/gt/bancoindustrial/pago-de-servicios-en-bi-en-linea",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Pago-de-servicios-BEL-Web.jpg",
          "subtitle": "Pasos para realizar pago de servicios desde Bi en Línea Web.",
          "title": "Pago de servicios desde Bi en Línea Web",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/pago-de-servicios-con-bi-en-linea-app",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Pago-de-servicios-BEL-App.jpg",
          "subtitle": "Pasos para realizar pago de servicios desde Bi en Línea App.",
          "title": "Pago de servicios desde Bi en Línea App",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
  ],
}
`;

exports[`Pagos servicio app 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Conoce cómo pagar tus servicios de forma inmediata como agua, luz , teléfono y a más de 2, 300 proveedores como universidades, colegios o mantenimiento del hogar.",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/pago-de-servicios-con-bi-en-linea-app",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Pago-de-servicios-BEL-App.jpg",
          "subtitle": "Pasos para realizar pago de servicios desde Bi en Línea App.",
          "title": "Pago de servicios desde Bi en Línea App",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": " https://www.corporacionbi.com/gt/bancoindustrial/pago-de-servicios-en-bi-en-linea",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Pago-de-servicios-BEL-Web.jpg",
          "subtitle": "Pasos para realizar pago de servicios desde Bi en Línea Web.",
          "title": "Pago de servicios desde Bi en Línea Web",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
  ],
}
`;

exports[`Pagos servicio web 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Conoce cómo pagar tus servicios de forma inmediata como agua, luz , teléfono y a más de 2, 300 proveedores como universidades, colegios o mantenimiento del hogar.",
    },
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": " https://www.corporacionbi.com/gt/bancoindustrial/pago-de-servicios-en-bi-en-linea",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Pago-de-servicios-BEL-Web.jpg",
          "subtitle": "Pasos para realizar pago de servicios desde Bi en Línea Web.",
          "title": "Pago de servicios desde Bi en Línea Web",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://www.corporacionbi.com/gt/bancoindustrial/pago-de-servicios-con-bi-en-linea-app",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Pago-de-servicios-BEL-App.jpg",
          "subtitle": "Pasos para realizar pago de servicios desde Bi en Línea App.",
          "title": "Pago de servicios desde Bi en Línea App",
        },
      ],
      "response_index": 1,
      "response_type": "carousel",
    },
  ],
}
`;
